app/src/test/java/com/infomaniak/swisstransfer/TransferSectionGroupingTest.kt

```diff
@@ -106,13 +109,13 @@ class TransferSectionGroupingTest {
                 // Case: data class with one parameter
                 val constructor = subClass.primaryConstructor
                 assertNotNull("This test needs to create instances of this class ${subClass.simpleName}", constructor)
-                assertEquals(1, constructor?.parameters?.count())
+                assertEquals(2, constructor?.parameters?.count())
 
-                val firstInstanceUid = constructor!!.call("aaa").uid
+                val firstInstanceUid = constructor!!.call(0, 0).uid
                 assert(firstInstanceUid !in previousUids)
                 previousUids.add(firstInstanceUid)
 
-                val secondInstanceUid = constructor.call("bbb").uid
+                val secondInstanceUid = constructor.call(1, 1).uid
```

Comment:
maybe, just to be sure, we could make it (1, 1) and (2, 2). Just because (0, 0) is actually illegal in the code and it would be a bother if we break a test in the futur by implementing logic that breaks when (0, 0) is provided even though 0 is only provided inside of a test case